Userland/Utilities/sed.cpp

```diff
@@ -942,27 +942,33 @@ ErrorOr<int> serenity_main(Main::Arguments args)
         pos_args.remove(0);
     }
 
+    HashMap<String, String> paths_to_unveil;
```

Comment:
It's fine as-is since the permission strings are short strings, but this could be:
```c++
HashMap<String, StringView>
```

Reply:
I actually tried that and it did not compile with StringView, no idea why exactly.

Reply:
:shrug: this patch compiles for me:
```c++
diff --git a/Userland/Utilities/sed.cpp b/Userland/Utilities/sed.cpp
index 9a7ac61c51..30ff078370 100644
--- a/Userland/Utilities/sed.cpp
+++ b/Userland/Utilities/sed.cpp
@@ -953,13 +953,13 @@ ErrorOr<int> serenity_main(Main::Arguments args)
         pos_args.remove(0);
     }
 
-    HashMap<String, String> paths_to_unveil;
+    HashMap<String, StringView> paths_to_unveil;
 
     for (auto const& input_filename : TRY(script.input_filenames())) {
-        TRY(paths_to_unveil.try_set(TRY(FileSystem::absolute_path(input_filename)), edit_in_place ? "rwc"_short_string : "r"_short_string));
+        TRY(paths_to_unveil.try_set(TRY(FileSystem::absolute_path(input_filename)), edit_in_place ? "rwc"sv : "r"sv));
     }
     for (auto const& output_filename : TRY(script.output_filenames())) {
-        TRY(paths_to_unveil.try_set(TRY(FileSystem::absolute_path(output_filename)), "w"_short_string));
+        TRY(paths_to_unveil.try_set(TRY(FileSystem::absolute_path(output_filename)), "w"sv));
     }
 
     Vector<File> inputs;
@@ -967,7 +967,7 @@ ErrorOr<int> serenity_main(Main::Arguments args)
         if (filename == "-"sv) {
             inputs.empend(TRY(File::create_from_stdin()));
         } else {
-            TRY(paths_to_unveil.try_set(TRY(FileSystem::absolute_path(filename)), edit_in_place ? "rwc"_short_string : "r"_short_string));
+            TRY(paths_to_unveil.try_set(TRY(FileSystem::absolute_path(filename)), edit_in_place ? "rwc"sv : "r"sv));
             auto file = TRY(Core::File::open(filename, Core::File::OpenMode::Read));
             if (edit_in_place)
                 inputs.empend(TRY(File::create_with_output_file(LexicalPath { filename }, move(file))));
```

Reply:
Strange.